### DISCUSSION TITLE
How do you use this blk extractor?

### DISCUSSION THREAD

**Discussion by Hadiiiiiiiiiiiiii** [Warthunder-Open-Source-Foundation/wt_ext_cli#4] [Questions]
Hey, i know this isn't really an issue but since there wasn't much in the Readme.md file and no release i was confused as to how to exactly use this tool, do you have a guide anywhere?

**Comment by FlareFlo** [comment]
It is currently very WIP. If you do want to try it out, look at [the reference CLI implementation](https://github.com/Warthunder-Open-Source-Foundation/wt_ext_cli). It too currently lacks proper readme docs, but i will write it up today.

**Comment by FlareFlo** [comment]
Documentation for the tool can be found here: https://github.com/Warthunder-Open-Source-Foundation/wt_ext_cli/blob/master/Readme.md

I will transfer this issue to said repo until any problems are cleared up.

**Comment by FlareFlo** [comment]
Converted this into a discussion.

**Comment by FlareFlo** [comment]
For future references, the manual and help should reflect capabilities.
https://github.com/Warthunder-Open-Source-Foundation/wt_ext_cli/blob/master/usage_manual.md
